### PR TITLE
Remove PWA

### PR DESCRIPTION
**Pull request by Soitora**
As I had hoped would not happen, people fall for the "Install app" message that comes up when PWA is active.

### CODE REVIEW

**Comment by Soitora** [comment]
Honestly not sure why this fails, local doesn't give any sort of error...

**Comment by arkon** [comment]
I cleared the cache and it still hits the same issue. Maybe try just removing your `node_modules` and `package-lock.json` and rebuilding it?

**Comment by specterflare** [comment]
I don't know if it's okay for me to comment here, but this PR breaks #202 (which was fixed by #204) again by shifting the content back to the left.

**Comment by Soitora** [comment]
> I don't know if it's okay for me to comment here, but this PR re-breaks #202 (which was fixed by #204) by shifting the content back to the left.

This should be incorrect, this PR is built prior to the sidebar fix, thus it will not display those changes in the Netlify preview. Nothing changed in this PR could break that.

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -7,17 +7,11 @@ module.exports = {
 
 	// prettier-ignore
 	head: [
-		// PWA headers
+		// Embed headers
 		['link', { rel: 'icon', href: '/favicon.ico' }],
 		['link', { rel: 'manifest', href: '/manifest.json' }],
```
Comment (arkon) [inline src/.vuepress/config.js:12]: Doesn't this need to be removed too?

Comment (Soitora) [reply src/.vuepress/config.js:12]: Manifest.json?
That's for the embeds and android shortcuts, it's stuff that I would use regardless of PWA

Comment (Soitora) [reply src/.vuepress/config.js:12]: Not sure if Discord has changed but last time I fixed embeds for websites it needed some stuff from Manifest, but that was more if you wanted to specify more embed information like big image, small image, widge image, thumbnail or photo etc

Comment (arkon) [reply src/.vuepress/config.js:12]: Ah, I see. Not very familiar with that stuff.

**Pull request merged**